spring-rabbit/src/main/java/org/springframework/amqp/rabbit/log4j2/AmqpAppender.java

```diff
@@ -237,96 +342,9 @@ public Message postProcessMessageBeforeSend(Message message, Event event) {
 		@Override
 		public void run() {
 			try {
-				RabbitTemplate rabbitTemplate = AmqpAppender.this.rabbitTemplate;
-				rabbitTemplate.setConnectionFactory(AmqpAppender.this.manager.connectionFactory);
 				while (true) {
```

Comment:
Hm... I wonder how is it going to stop looping?..

I see that we perform `senderPool.shutdownNow()` and don't wait for finishing, but that really looks for me that our `EventSender` will never stop.